src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -142,9 +143,6 @@ func Run(updateHandler func(updated *Config)) error {
 
 func updateGlobals(cfg *Config) error {
 	globals.InstanceId = cfg.InstanceId
-	loc := &geolookup.City{}
-	loc.Country.IsoCode = cfg.Country
-	globals.SetLocation(loc)
```

Comment:
This seems like it was a bug. Not sure why we ever set the global location here, but in practice it also frequently would *overwrite the location* discovered in our geolookup -- i.e. the likely cause of the many "xx" locations we see in analytics.

This change also makes it so that the only thing that ever modifies the location is the geolookup itself, which I think makes sense.